static/js/modules/tables.js

```diff
@@ -320,10 +309,17 @@ function filterSuccessUpdates(changeCount) {
       clearTimeout(messageTimer);
     }
 
-    $label.after($('<div class="filter__message filter__message--success">' + message + '</div>')
-      .hide().fadeIn());
+    // Clicking on "Clear all filters" will remove all dropdown checkboxes,
+    // so we check to make sure the message isn't shown inside the dropdown panel.
+    if (!$label.parent().parent().hasClass('dropdown__list')) {
```

Comment:
What about using `$label.closest('.dropdown__list').length < 1` to be a little more concise?